WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -165,6 +164,32 @@ public struct WordPressAuthenticatorConfiguration {
 }
 
 
+// MARK: - WordPress Authenticator Styles
+//
+public struct WordPressAuthenticatorStyle {
+    /// Alternative logins link color
+    ///
+    let linkColor: UIColor
+
+    /// Alternative logins highlight color
+    ///
+    let highlightColor: UIColor
```

Comment:
Please set `linkColor` and `highlightColor` as *public*, otherwise they won't be accessible from other frameworks.

Reply:
✔️